.conform.yaml

```
@@ -0,0 +1,14 @@
+policies:
+  - type: commit
+    spec:
+      conventional:
+        types:
+          - chore
+          - ci
+          - docs
```

```suggestion
          - docs
```
Stemmer her for `doc` i stedet. Mest for at det er det jeg har brukt de siste årene.

---

Stemmer for `docs` da det fremstår for meg som mer etablert, og trekkes frem som eksempel i docsa:

Nr 4 her: https://www.conventionalcommits.org/en/v1.0.0/#summary